contracts/src/interfaces/IEigenDACertVerifierV2.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDAThresholdRegistry} from "./IEigenDAThresholdRegistry.sol";
+import "./IEigenDAStructs.sol";
+
+interface IEigenDACertVerifierV2 {}
```

Comment:
Did you not add any functions here because the signatures to use are still slated to change?

Reply:
Yes, I basically just wanted to make a place for the interface without actually defining it yet.